sources/packages/web/src/components/fa-application/ContentGroup.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
This component should be in global scope....

Reply:
Moved!